cli/main.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+from cli.app import CONF, IRFactory
+
+
+def main_provision():
```

Comment:
instead of a dedicated method per spec, just loop over a settings value (mentioned in a previous comment) and it just under "__main__"